labelImg.py

```diff
@@ -1141,6 +1141,7 @@ def resizeEvent(self, event):
     def paintCanvas(self):
         assert not self.image.isNull(), "cannot paint null image"
         self.canvas.scale = 0.01 * self.zoomWidget.value()
+        self.canvas.labelFontSize = int(0.02 * max(self.image.width(), self.image.height()))
```

Comment:
why is 0.02?

Reply:
Cloud you provide the before and after screenshots?

Reply:
0.02 is just an empirically set value by me, for proper visibility.

Please see below for the before & after:

1. Before the modification, the label is not visible on a cell-phone taken 4k*3k photo. We have to zoom in to see it.

![before](https://user-images.githubusercontent.com/12696742/103451179-b27d7200-4cfb-11eb-8834-fddf797144de.jpg)

![before_zoomed](https://user-images.githubusercontent.com/12696742/103451181-b9a48000-4cfb-11eb-93aa-71dd4e370a28.jpg)

2. After the modification, the label is visible without zooming. 

![after](https://user-images.githubusercontent.com/12696742/103451183-bad5ad00-4cfb-11eb-85b4-fe3048bcb1b2.jpg)